bin/bi-registry

```diff
@@ -202,6 +203,78 @@ parse_params() {
     return 0
 }
 
+get_tags_for_image() {
+    local image="${1}"
+    local last=""
+    local all_tags
+    local registry
+    local repo
+    registry=$(echo "$image" | cut -d'/' -f1)
+    repo=$(echo "$image" | cut -d'/' -f2-3)
+
+    while :; do
+        local tags=()
+        # query api and add tags to local array
+        while IFS='' read -r tag; do
+            tags+=("$tag")
+        done < <(
+            curl --silent "https://${registry}/v2/${repo}/tags/list?n=100&last=${last}" |
+                jq -r '.tags[]'
+        )
+
+        # if array is empty, we've got all the tags
+        [[ "${#tags[@]}" -eq 0 ]] && break
+
+        # otherwise add the local tags to the outer array
+        all_tags+=("${tags[@]}")
+        # and update the "last" parameter
+        last="${tags[-1]}"
+    done
+
+    echo "${all_tags[@]}"
+}
```

Comment:
This functionality is already in the registry-tool. What do you think about using the registry list for this.

So we have

battery_included_keycloak & keycloak in the registry. Every night, the registry will be updated, which means that sometimes the registry will have a new tag for Keycloak that's not present for battery_included_keycloak. We can trigger a new workflow whenever the set of tags for the two registry keys is not the same.